tests/test_butler.py

```diff
@@ -1798,6 +1798,14 @@ def testTransferUuidToUuid(self):
         # Setting id_gen_map should have no effect here
         self.assertButlerTransfers(id_gen_map={"random_data_2": DatasetIdGenEnum.DATAID_TYPE})
 
+    def _enable_trust(self, datastore) -> None:
+        if hasattr(datastore, "trustGetRequest"):
+            datastore.trustGetRequest = True
+        elif hasattr(datastore, "datastores"):
+            for datastore in datastore.datastores:
+                if hasattr(datastore, "trustGetRequest"):
+                    datastore.trustGetRequest = True
```

Comment:
Could chained datastore include other chained datastores?

Reply:
I hope not. I don't think anything stops it though.

Reply:
It maybe safer to make this method to recursively call itself? But for tests it probably does not matter.

Reply:
Right, if a test fails because somehow I've got a chained datastore inside a chained datastore I can fix the test.